Persephone/Components/Browser/Album Browser/AlbumViewItem.swift

```diff
@@ -57,16 +59,18 @@ class AlbumViewItem: NSCollectionViewItem {
     }
   }
 
-  func setAppearance() {
+  func setAppearance(selected isSelected: Bool) {
     if #available(OSX 10.14, *) {
       let darkMode = NSApp.effectiveAppearance.bestMatch(from:
         [.darkAqua, .aqua]) == .darkAqua
 
       albumCoverView.layer?.borderColor = darkMode ? .albumBorderColorDark : .albumBorderColorLight
-      albumCoverBox.layer?.borderColor = NSColor.controlAccentColor.cgColor
+      albumCoverBox.layer?.borderColor = isSelected ? NSColor.controlAccentColor.cgColor : CGColor.clear
+      albumCoverBox.layer?.backgroundColor = isSelected ? NSColor.controlAccentColor.cgColor : CGColor.clear
```

Comment:
Not sure if this is desirable, but it seems like you could do `albumCoverBox.layer?.backgroundColor = albumCoverBox.layer?.borderColor`